tests/api/routes/test_projects.py

```diff
@@ -0,0 +1,75 @@
+from collections.abc import Generator
+from unittest.mock import MagicMock, Mock
+
+import pytest
+from fastapi.testclient import TestClient
+from tad.schema.project import ProjectNew
+from tad.schema.system_card import SystemCard
+from tad.services.instruments import InstrumentsService
+from tad.services.storage import FileSystemWriteService
+
+from tests.constants import default_instrument
+
+
+@pytest.fixture()
+def init_instruments() -> Generator[None, None, None]:  # noqa: PT004
```

Comment:
This technique is very nice! Where possible, we should apply this more when using Mocks for "temporary original replacements".